docs/production-deployment.rst

```diff
@@ -191,14 +208,14 @@ to 20::
     helm upgrade anonlink entity-service --namespace=testing --set workers.replicas="20"
```

Comment:
a few lines before you used `data61/entity-service`. Now only `entity-service`. Is that deliberate?

Reply:
Will change - the issue is because previously all the docs assumed you were deploying from source code.